src/05-internship-interviews.md

```diff
@@ -1,5 +1,87 @@
+> 
+> I like to allow for 45 minutes for the code review and 15 minutes for any follow-up questions you may have for me. Let me know when you are available this week.
+> 
+> Regards,
+> <your name>
+> Introduction
+
+I’ve been saying something along these lines at the start of the interview. I’m trying to clarify what I’m looking for right at the start.
+
+> We're going to be doing a code review. The goal is for you to explain the context and behavior of your application to me. I'm looking for you to help me comprehend your code base and your thought process arriving at this point, not judging your approach. I liken it to, we're not approaching the code base like an algorithms course would, looking for the big-o notation of an algorithm. Rather, we’re interested in what the algorithm does and, probably more importantly, why it does it. I'll have a lot of “what does this do?”, “can you tell me more about that?”, and “can we take a look at that method?”, kind of questions.
+
+Pause here to see if they have any questions. If not, have the candidate start by sharing their screen and editor. Ask them to give you a high level context of what the application is, what the problem is and where you will be starting in the application.
+
+Resist the urge to fix their code. This doesn’t really tell us anything about the candidate. If you see something like a race condition or a bug, pose the question for how they would handle the buggy scenario.
+
+Follow along with the code. Ask them to stop and explain things you find interesting or confusing. See where they take you. Example, seeing a dbConnect parameterized query you can ask how that library works “is that string and value in an array a parameterized query?”. See if they identify the term and can make the leap to SQL escaping/SQL injection.
+
+Joke, praise, have fun. This works best if the candidate is relaxed and is enjoying showing off their code.
+
+### Quantification
+
+Picking an arbitrary 1-5 scale here. We can adjust if we find we need more nuance when quantifying a category. General ranking is 1 - poor, 3 - average, 5 - exceptional with 2/4 for shades of gray in between.
+
+#### Overall
+
+Would you want to mentor/work with the candidate? Use this section as an overall rating of how well the candidate did and the following sections as the details.
+ 1. present (Something didn’t sit right with me)
+ 1. likeable
+ 1. technical (Seem like a nice person)
+ 1. articulate
+ 1. passionate (Please put them on my team yesterday)
+
+#### Codebase
+
+What were you reviewing?
+
+#### Communication
+
+How well does the candidate communicate their understanding of the code to you.
+
+ 1. candidate either doesn’t understand their code or can’t explain what is happening beyond existing comments
```

Comment:
Are these meant to be blank lines? it's incrementing the number in the md render if you meant to leave them for responses

Reply:
Yes, I wanted a 1-5 bulled list for the ranking. Its more of an example than a fill the blank template